synapse/lib/stormlib/stix.py

```diff
@@ -78,7 +87,7 @@ def uuid4(valu=None):
                         ('attributed-to', 'identity', ''),
                         ('located-at', 'location', ':hq -> geo:place'),
                         ('targets', 'identity', '-> ou:campaign -> risk:attack :target:org -> ou:org'),
-                        # ('targets', 'location', ''),
+                        ('targets', 'vulnerabilty', '-> ou:campaign -> risk:attack -> it:prod:softver -> risk:hasvuln -> risk:vuln'),
```

Comment:
there are additional pivots through risk:attack which can be added

Reply:
This pivot is incorrect. this would report that a threat actor targets all vulns in any software they attack.